contents/blog/raising-3m-for-os.md

```diff
@@ -23,44 +23,97 @@ So, what are your options for building a company around the project?
 
 ## Venture Capital (VC)
 
-VC means revenue won't bottleneck your growth. That's pretty much the advantage. Depending very heavily on the firm, you may get a lot of support with meeting early customers, hiring, free VC swag and your strategy.
+VC means revenue won't bottleneck your project's growth (usage or features), if you spend the money wisely. That's pretty much the advantage. You will obviously have to build a company around the project so you have a way of making money at least in the long run.
```

Comment:
`(usage or features)` doesn't make sense to me

Reply:
Clarified.